src/main/webapp/js/countryCodes.es6

```diff
@@ -430,6 +431,7 @@ const codeToCountryName = {
     'OMN': 'Oman',
     'PAK': 'Pakistan',
     'PLW': 'Palau',
+    'PSE': 'Palestine',
     'PSE': 'Palestinian Territory, Occupied',
```

Comment:
There's a duplicate key here. You can remove the previous one (`Palestinian Territory, Occupied`).